include/deal.II/base/vectorization.h

```diff
@@ -426,14 +426,20 @@ class VectorizedArray
    */
   using value_type = Number;
 
-  static_assert(width == 1,
-                "You specified an illegal width that is not supported.");
+  /**
+   * TODO
+   */
+  static constexpr bool is_available = false;
 
   /**
    * Default empty constructor, leaving the data in an uninitialized state
    * similar to float/double.
    */
-  VectorizedArray() = default;
+  VectorizedArray()
+  {
+    static_assert(width == 1,
+                  "You specified an illegal width that is not supported.");
```

Comment:
You'll have to copy this into every one of the constructors then.

Reply:
I had to think about this constructor for a while, because we want the `VectorizedArray` class to be of standard layout to ensure it works well with memcpy and other basic (C-style) operations. I had to convince myself about `std::is_trivial` and `std::is_standard_layout` and see that we want the latter and that this user-defined constructor is fine with that desire (in fact, we already had user-provided constructors before, so nothing really changes here). In that sense :+1: from my end.